.github/workflows/nightly.yml

```diff
@@ -67,7 +67,8 @@ jobs:
         token: ${{ secrets.SECRET_TOKEN }}
         repository: f3d-app/f3d
         tag: nightly
-        assets: "F3D-*-${{ matrix.enable_raytracing == 'true' && 'raytracing' || null }}-${{matrix.config.name}}*"
+        assets: "F3D-*-${{ matrix.enable_raytracing == true && 'raytracing-' || null }}${{matrix.config.name}}*"
+        fail-if-no-assets: false
```

Comment:
Is it intended?

Reply:
yes it is, as the asset name changed, I think it is better to not fail in that case.